ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/visitor/classes/JavaClassIdentityCollectorTest.java

```diff
@@ -0,0 +1,75 @@
+package edu.usb.argos.ASTProcessor.visitor.classes;
+
+import edu.usb.argos.ASTProcessor.antlr.JavaLexer;
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.classes.AnnotationInfo;
+import edu.usb.argos.ASTProcessor.visitor.core.services.collectors.classes.JavaClassIdentityCollector;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JavaClassIdentityCollectorTest {
+    private JavaClassIdentityCollector classIdentityCollector;
+    private CommonTokenStream tokenStream;
+    private JavaParser parser;
+    private JavaParser.CompilationUnitContext compilationUnit;
+
+    @BeforeEach
+    void setUp() {
+        String testClass =
+                "package com.example;\n" +
+                        "\n" +
+                        "import java.util.List;\n" +
+                        "\n" +
+                        "@TestAnnotation(value = \"test\")\n" +
+                        "public class TestClass extends BaseClass implements Interface1, Interface2 {\n" +
+                        "    private String field;\n" +
+                        "    public void method() {}\n" +
+                        "}";
+
+        CharStream input = CharStreams.fromString(testClass);
+        JavaLexer lexer = new JavaLexer(input);
+        tokenStream = new CommonTokenStream(lexer);
+        parser = new JavaParser(tokenStream);
+        compilationUnit = parser.compilationUnit();
+        classIdentityCollector = new JavaClassIdentityCollector();
+    }
+
+    @Test
+    void getClassName_ShouldReturnCorrectClassName() {
```

Comment:
Use CamelCase for test name, don't snake test and the same with the rest

Reply:
Done